sdk/rust/oak/src/grpc/mod.rs

```diff
@@ -139,6 +139,35 @@ pub trait OakNode {
     fn invoke(&mut self, method: &str, req: &[u8], writer: ChannelResponseWriter);
 }
 
+impl<T: OakNode> crate::Node<Invocation> for T {
+    /// Perform a gRPC event loop for an [`OakNode`].
+    ///
+    /// Invoking the given `node`'s [`invoke`] method for each incoming request that
+    /// arrives on the inbound channel as a serialized [`Invocation`] object,
+    /// giving the [`invoke`] method the outbound channel for encapsulated responses
+    /// to be written to.
+    ///
+    /// [`invoke`]: OakNode::invoke
+    fn handle_command(&mut self, command: Invocation) -> std::result::Result<(), crate::OakError> {
+        let invocation = command;
```

Comment:
Could just name the parameter `invocation`?

Reply:
Done.